src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipeflatlands.java

```diff
@@ -10,7 +10,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.SENIOR_ADMIN, source = SourceType.ONLY_CONSOLE, blockHostConsole = true)
-@CommandParameters(description = "Wipe the flatlands map. Requires manual restart after command is used.", usage = "/<command>")
+@CommandParameters(description = "Wipe the flatlands map. Server will restart after command is used.", usage = "/<command>")
```

Comment:
Not sure this is true..?

Reply:
It is, server forces a restart in order to wipe it